cmd/ocsp-updater/main.go

```diff
@@ -51,28 +110,69 @@ func setupClients(c cmd.Config, stats statsd.Statter) (rpc.CertificateAuthorityC
 	return cac, closeChan
 }
 
-func (updater *OCSPUpdater) processResponse(tx *gorp.Transaction, serial string) error {
-	certObj, err := tx.Get(core.Certificate{}, serial)
-	if err != nil {
-		return err
-	}
-	statusObj, err := tx.Get(core.CertificateStatus{}, serial)
-	if err != nil {
-		return err
+var errNoStaleResponses = errors.New("No stale responses to update")
+
+func (updater *OCSPUpdater) findStaleOCSPResponses(oldestLastUpdatedTime time.Time, batchSize int) ([]core.CertificateStatus, error) {
+	var statuses []core.CertificateStatus
+	_, err := updater.dbMap.Select(
+		&statuses,
+		`SELECT cs.*
+		 FROM certificateStatus AS cs
+		 JOIN certificates AS cert
```

Comment:
I see this join didn't get removed. I'm guessing you're saying we should do this in another PR that is maybe not for GA? If so, mind filing a ticket for it?

@jsha, does that sound like an okay plan?